packages/SketchMorph2-Core.package/M2Backend.class/instance/drawTriangle.st

```diff
@@ -1,8 +1,9 @@
 tool selection
 drawTriangle
-	self pen sourceForm: self prepareTrianglePenSource.
+	self pen sourceForm: (self prepareTrianglePenSource offset: (self penNibOffset: self brushSize asPoint)).
 	self
 		isErasing: false;
 		mode: #pen;
 		currentTool: #drawTriangle;
+		currentCursor: ((Form extent: self brushSize asPoint depth: self defaultDepth) fillShape: self prepareTrianglePenSource fillColor: self currentColor);
```

Comment:
Maybe we should add a cursor for that like we did for `drawEraser` to make the code more consistent.